portfolio/src/main/webapp/ceramics.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Jag's Ceramic Creations</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <div id="content">
+      <h1>Some Wheel-Thrown Pieces</h1>
+      <div class="image-thumbnails">
+        <a href="images/stanley-1.jpg"><img src="/images/ceramics.jpg"/></a>
```

Comment:
did you still want the stanley-1.jpg image? I don't think you have it in your images file

Reply:
deleted the stanley image.